scripts/release-update.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env sh
```

Comment:
why is this file here? it's a duplicate of #11716 right?

Reply:
Yes, this was wrongly commited here. I removed it in the last commit aba20e4, it's not part of the merge diff.